src/pages/basket/Basket.tsx

```diff
@@ -27,6 +31,10 @@ export const Basket = () => {
         setBooks(prevBooks => prevBooks.filter(book => 
             booksLocalStore.find(localBook => book.idBook === localBook.idBook && localBook.quantity )))
     }, [booksLocalStore])
+    
+    const goBackPage = useCallback(() => 
+        navigate(-1)
```

Comment:
if we will open this page, we won't navigate history

Reply:
discused